docs/csv/example_files/ballot_selection.txt

```diff
@@ -1 +1,6 @@
-id,sequence_order
\ No newline at end of file
+id,sequence_order
```

Comment:
Correct. This file is not used in the finished documentation. Will remove the file.